text/0000-rand-crypto.md

```diff
@@ -0,0 +1,192 @@
+- Feature Name: rand-crypto
+- Start Date: 2017-08-18
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Background
+
+This RFC is an offshoot of the [Rand revision RFC], on the subject of:
+
+*   splitting rand into multiple crates
+*   splitting the current `Rng` trait into a crypto-oriented trait and
+    numerical-application-oriented trait
+*   impl of above trait(s) for OS-provided randomness
+*   naming conventions for above things
+
+# Summary
+[summary]: #summary
+
+Create a new crate, `crypto-rand`, with only the following contents:
+
+```rust
+pub enum Error {
+    pub Unspecified,
+}
```

Comment:
Not sure if we need `enum` here. Can't we just use `pub struct Error;` ? @briansmith can you provide motivation for using `enum`? I think it's quite unlikely we'll need to expand this error in the future.

Also what about implementing `Error` trait (btw conflict of the names) and deriving others, e.g. `Copy`, `Clone`, `Debug`, `Display`, etc.? For `Error` we'll need to introduce `std` feature gate.